.circleci/config.yml

```diff
@@ -7,6 +7,7 @@ version: 2.1
 # Orb declarations
```

Comment:
Nitpick, but this doesn't seem pertinent to the task described in the PR.  Would be awesome to have additional tasks like these spun off into separate PR's just for the sake of cutting down on overall size and making review easier.

Reply:
No problem, i'll keep that in mind next time for tasks like these